packages/p2p/src/components/modal-manager/modals/buy-sell-modal/buy-sell-modal.tsx

```diff
@@ -98,7 +98,7 @@ const BuySellModal = () => {
                     submitForm,
                     values: {
                         currency: buy_sell_store.account_currency,
-                        input_amount: buy_sell_store.form_props.input_amount.toFixed(2),
+                        input_amount: Number(buy_sell_store.form_props.input_amount).toFixed(2),
```

Comment:
toFixed only works for number type, and input_amount was type string, this caused issues if the user entered a value into the input_amount field. This is also happening in prod, but should be fixed here